src/csharp/Microsoft.Spark/Utils/TypeConverter.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Spark.Utils
+{
+    /// <summary>
+    /// Converts one type to another.
```

Comment:
Can you put a more detailed summary?

Reply:
updated summary.

Reply:
`Converts one type to another.` is a bit misleading since most of the time, it's just a pass thru.

Reply:
ok.  I'll remove it.